src/App.js

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React from "react";
+
+let todoList = [];
```

Comment:
Although your code works, it is common convention to keep all variables and functions related to a component inside of the component, regardless if it is a functional component or class-based component. We will run across an exception to this later in the course. But for now,I recommend placing your todoList object before your return statement.

Reply:
Placed the todoList object before the return statement.
Thank you!